hadoop-hdds/docs/content/concept/StorageContainerManager.md

```diff
@@ -46,7 +46,7 @@ replicas. If there is a loss of data node or a disk, SCM
 detects it and instructs data nodes make copies of the
 missing blocks to ensure high availability.
 
- 3. **SCM's Ceritificate authority** is in
+ 3. **SCM's Certificate authority** is in
```

Comment:
Should it be both upper case or lower case for the first character? 
E.g. either `Certificate Authority` or `certificate authority`?

Reply:
Thanks @amaliujia for the suggestion. Both names sound good to me.
I'll update the patch based on your suggestion.